src/lib/execution_plan.rs

```diff
@@ -168,6 +168,12 @@ fn create_workspace_task(crate_info: CrateInfo, task: &str) -> Task {
             make_line.push_str(&log_level);
             make_line.push_str(" --env CARGO_MAKE_CRATE_CURRENT_WORKSPACE_MEMBER=");
             make_line.push_str(&member_name);
+
+            if envmnt::is_or("CARGO_MAKE_PASS_PROFILE", false) {
```

Comment:
how about
* always pass the profile in the cli - this will require you to update the tests, however anyway i would like a test for this
* if env is true pass the same one as you do now, if false, pass the default profile. this will make things more consistent. I could later on, default the env to true in the internal makefile and publish a backward breaking version.
* can we rename to CARGO_MAKE_USE_WORKSPACE_PROFILE - helps to understand this is workspace related.

Reply:
Done.
I really need some guidance on making a test here, I played around a bit, but not really sure how I can make an effective test.